app/components/subject-viewer.cjsx

```diff
@@ -102,12 +102,13 @@ module.exports = createReactClass
 
   componentDidUpdate: (prevProps) ->
     isGroupSubject = @props.subject?.metadata?['#subject_group_id']?
+    allowFlipbook = @props.allowFlipbook || isGroupSubject
```

Comment:
Should this be `!isGroupSubject` or was the previous behaviour wrong?

Reply:
I think that would re-introduce the bug: turn on the flipbook for all subjects that are not group subjects.

The staging link to GZ: Weird and Wonderful seems to work, so I'm inclined to leave this for now.